### DISCUSSION TITLE
Debugging capacitor IOS app on chrome://inspect

### DISCUSSION THREAD

**Discussion by workstreamautomation** [ionic-team/capacitor#4729] [Ideas]
Hello Great people i am working on a capacitor app in which i am using SAPUI5 typescript js and have successfully build an app for android now i am working on ios build , now i am facing issues while debugging capacitor app because my ios app gui is not working properly like in android so any suggestions for debugging.![image](https://user-images.githubusercontent.com/85283211/122223672-d3620a00-cecc-11eb-9271-c7cbc18324cc.png)![image](https://user-images.githubusercontent.com/85283211/122223770-ec6abb00-cecc-11eb-9810-2564f9462d60.png)

**Comment by jcesarmobile** [comment]
I think it can be done using [this proxy](https://github.com/google/ios-webkit-debug-proxy)

But it's easier to just use Safari, if you enable the Developer menu, all your devices and simulators will appear there and you can debug in a similar way than with chrome inspector.

https://www.browserstack.com/guide/how-to-debug-on-iphone

**Reply by workstreamautomation** [reply]
@jcesarmobile  Thanks you soo much.i have tried debugging in safari but i can't find developer resources i have to use sourcemap toogle option just like chrome but seems like safari doesn't have that functionality.

**Reply by jcesarmobile** [reply]
I’ve heard good things about https://inspect.dev/, but it’s not free, you can try it and see it it’s worth the price

**Reply by workstreamautomation** [reply]
@jcesarmobile thanks, its really good but I wan't to disable the load http developer resources through target option just like chrome but this option seems to be in their backlog hopefully in future we can use it but not right now. https://github.com/inspectdev/inspect-issues/issues/3
